src/scripts/quests/Quest.ts

```diff
@@ -19,7 +19,7 @@ abstract class Quest {
         this.amount = amount;
         const randomPointBonus = 0.9 + SeededRand.next() * 0.2; // random between 0.9 and 1.1
         this.pointsReward = Math.ceil(pointsReward * randomPointBonus);
-        this.xpReward = pointsReward / 10;
+        this.xpReward = pointsReward / 8;
```

Comment:
Maybe we should have a base exp reward instead of increasing all the quest exp by 25%,
This will make completing the lower point quest slightly more worthwhile.
```suggestion
        this.xpReward = 100 + (pointsReward / 10);
```